code/modules/paperwork/photocopier.dm

```diff
@@ -439,7 +550,7 @@ GLOBAL_LIST_INIT(paper_blanks, init_paper_blanks())
 		return null
 	var/obj/item/photo/copied_pic = new(src, photo.Copy(photo_color == PHOTO_GREYSCALE ? TRUE : FALSE))
 	delete_paper(PHOTO_PAPER_USE)
-	toner_cartridge.charges -= PHOTO_TONER_USE
+	toner_cartridge.charges -= PHOTO_TONER_USE / toner_efficiency
```

Comment:
I see a lot of repeated
```dm
toner_cartridge.charges -= BASE_AMOUNT_DEFINE / toner_efficiency
```
maybe better to move it under a `use_toner(base_amount)` proc, so the calculation for toner efficiency is only in one place

Reply:
Applied via https://github.com/tgstation/tgstation/pull/91258/commits/7c92685df75204ad97c4a068a17c12d5539dba6d